rules/parser.y

```diff
@@ -235,6 +237,22 @@ extra_labels_opts  :
                      { $$ = true }
                    ;
 
+vector_matching    :
```

Comment:
For clarity, please add an explicit `/* empty */` comment here, as in the other cases in this file where there's an empty rule.